src/AndroidDebugLauncher/Telemetry.cs

```diff
@@ -40,6 +40,11 @@ public enum LaunchFailureCode
             RunAsFailure,
 
             /// <summary>
+            /// run-as returned 'Package 'bla' is not debuggable'
```

Comment:
We should change it then as we may open source this in the future. I suggest using Squid but I'm a weirdo